adventureGame/AdventureGame/src/Main.py

```diff
@@ -1,2 +1,44 @@
 # -*- coding: utf-8 -*-
 
+import time
+
+def main():
+    titleScreen()
+    introduction()
+        
+def titleScreen():
+    print("|==============================|")
+    print("|  Welcome to Adventure Game!  |")
+    print("| Enter any value to continue! |")
+    print("|==============================|")
+    input()
+    for i in range(50):
+        print("\n")
+    
+def introduction():
+    print("Sir Gregos IV: \"Quickly, in here! They're too many of them!\"\n")
+    time.sleep(3)
+    print("You and your friend hurry inside the ominous room where sir Gregos IV awaits, closing the door behind you.\n")
```

Comment:
Change print text to:
"You and your friend hurry inside the ominous room where Sir Gregos IV awaits, close the door behind you.\n

Reply:
perhaps it should be: print("You and your friend hurry inside the ominous room where Sir Gregos IV awaits, and he closes the door behind you.\n) as what you said doesn't make a lot of sense to me

Reply:
My mistake. I mistook the third person text as speech from Sir Gregos. your original print statement was fine.